src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    internal partial class SolutionState
+    internal partial class SolutionCompilationState
```

Comment:
i didn't rename the files to avoid any github diff issues.  i plan to in followup.

Reply:
Do you have a follow-up list actually going somewhere for these?